src/BenchmarkDotNet/Configs/ConfigExtensions.cs

```diff
@@ -110,6 +111,7 @@ public static class ConfigExtensions
         [Obsolete("This method will soon be removed, please start using .AddLogicalGroupRules() instead.")]
         [EditorBrowsable(EditorBrowsableState.Never)] public static IConfig With(this IConfig config, params BenchmarkLogicalGroupRule[] rules) => config.AddLogicalGroupRules(rules);
         [PublicAPI] public static ManualConfig AddLogicalGroupRules(this IConfig config, params BenchmarkLogicalGroupRule[] rules) => config.With(c => c.AddLogicalGroupRules(rules));
+        [PublicAPI] public static ManualConfig AddEventHandler(this IConfig config, params IEventHandler[] eventHandlers) => config.With(c => AddEventHandler(c, eventHandlers));
```

Comment:
@adamsitnik , @caaavik-msft

> nit: it's unlikely that the users will need more than one instance

While this may be true, this makes it the only one of the `Add...` methods that doesn't accept a params array. Personally, I would prefer the API consistency.